addons/hr_holidays/static/src/components/month_specific_days/month_specific_days.js

```diff
@@ -0,0 +1,40 @@
+import { registry } from "@web/core/registry";
+import { SelectionField, selectionField } from "@web/views/fields/selection/selection_field";
+
+export class MonthSpecificDays extends SelectionField {
+    static props = {
+        ...SelectionField.props,
+        dependFieldName: String,
+    };
+
+    get options() {
+
+        let days = this.props.record.fields[this.props.name].selection;
+        const dependFieldName = this.props.dependsOn;
+        const dependValue = this.props.record.data[dependFieldName];
+
+        let lastDay;
+        if (dependValue == '2') {
+            lastDay = '29';
+        } else if (['4', '6', '9', '11'].includes(dependValue)){
```

Comment:
```suggestion
        } else if (['4', '6', '9', '11'].includes(dependValue)) {
```
Missing space before '{'